examples/golang/README.md

```diff
@@ -0,0 +1,32 @@
+# Upload Object to Ceph Cluster using AWS sdk
+
+This golang example shows us how to use a ceph cluster object gateway in place of an aws s3. A ceph cluster gateway is directly compatible with aws sdk or cli. So, you don't need to worry about rewriting your codebase. In more examples to come, you would be shown how to create and get notifications on a particular bucket and every other thing you can do with your aws cli or sdk
+
+To understand it further, build the app: 
+    
+    $go build add-object.go
+
+Then run the built binary with the value for the flags.
+
+Make sure you have created an object bucket on your ceph cluster
+
+You need to use this flags while running the built binary:
+
+    objectPath: Provide an object file path
+    bucketName: Provide the upload bucket path `example`
+    endpoint: Provide Ceph rgw endpoint for cluster
+    accessId: Provide your Ceph cluster rgw access id
+    accessKey: Provide your Ceph cluster rgw access key
+
+**Example:**
+
+    $./add-object --endpoint https://localhost:8888 --accessId "jlsdjfladfjs" --accessKey "lkasdljsdfjdf2345ajkhfahdfad3423lksdjf" \
```

Comment:
* please use the default parameters from the vstart tun:
```
S3 User Info:
  access key:  0555b35654ad1656d804
  secret key:  h7GhxuBLTrlhVUyxSPUKUV8r/2EI4ngqJxD7iBdBYLhwluN30JaT3Q==
```
* also, vstart is running the RGW on port 8000 and not 8888.
* last, why did you define "https"?

Reply:
I wanted to add some creativity, I make changes.